tests/phpunit/tests/rewrite/addRewriteEndpoint.php

```diff
@@ -25,11 +25,6 @@ public function setUp() {
 		$this->qvs = $GLOBALS['wp']->public_query_vars;
 	}
 
-	public function tearDown() {
-		$GLOBALS['wp']->public_query_vars = $this->qvs;
```

Comment:
The `$GLOBALS['wp']` variable is not being reset in `$GLOBALS['wp']::tearDown()`, so why can this be removed ?

Also: if it really _can_ be removed, the private `$qvs` property can also be removed, just like the assignment to that property in `setUp()`.

Reply:
The `$wp` global [gets reset in the base `tearDown()` method here](https://github.com/johnbillion/wordpress-develop/blob/429ff147337e13d517a2e45b8b1d1eacf8ed3e69/tests/phpunit/includes/abstract-testcase.php#L152), making this redundant.

Good point about the `$qvs` property, I thought this was being used in the `test_should_not_register_query_var_if_query_var_param_is_false()` test but it's not, it's a local variable. I'll get it removed.